recipes/default.rb

```diff
@@ -207,6 +207,8 @@
 service "kafka" do
   action [:enable, :start]
   supports :status => true, :restart => true
+  retries 2
+  retry_delay 5
```

Comment:
Do you think its worth making this configurable?

Reply:
I didn't think so - if these were Chef's defaults (rather than the actual 0 retries, 2 seconds apart) I think we would be fine with that. There are some configurable sleep times in the init.d script which should cover most transient issues already.